source/chapters/library.rst

```diff
@@ -312,8 +312,8 @@ Tracks can be loaded into a deck in several ways:
   and :guilabel:`Load in Deck 2`, among others. Making either selection will
   load a track into a deck.
 * By :ref:`control-keyboard` to load the selected track in the library track
-  table. Use keyboard :kbd:`Up` and :kbd:`Down` arrow button to select a track, :kbd:`Shift` + :kbd:`Left`
-  to load into Deck 1 and :kbd:`Shift` + :kbd:`Right` to load into Deck 2.
+  table. Use keyboard :kbd:`↑` and :kbd:`↓` arrow button to select a track, :kbd:`Shift` + :kbd:`←`
```

Comment:
Please revert that since we agreed to keep the literal key names because there are obviously no icons that work reliably on all platforms.
Sorry for the back and forth.

Reply:
I thought we just said we use the regular arrows instead of the more obscure ones.